content/guides/kubernetes/storage-integration.md

```diff
@@ -105,7 +105,7 @@ Running trident in a cluster will provision volumes against its supported
 providers, such as NetApp's SolidFire. It also handles concerns around backup
 and recovery. Another common project that follows this model is
 [rook](https://rook.io), which provides integration with providers like Ceph and
-NFS.
+NFS. Also, Dell EMC Project [Karavi](https://github.com/dell/karavi-observability) enriches CSI with enterprise storage capabilities such as Authorization, Resiliency and others.
```

Comment:
Main url to the project is : https://github.com/dell/karavi

Also, we have been told to name the project **Container Storage Module** and not Karavi anymore... Maybe we can rephrase as:
> Also, Dell EMC [Container Storage Modules](https://github.com/dell/karavi) (formerly known as Karavi) enriches CSI with...